docker-compose-prod.yml

```diff
@@ -1,14 +1,15 @@
 # Copyright © 2018 DIG Development team. All rights reserved.
 
-version: '3'
+version: '3.0'
 services:
     node:
         build:
             context: .
-            dockerfile: Dockerfile
-        volumes:
-            - ./:/var/project
-        entrypoint: /usr/local/bin/run.sh
+            dockerfile: docker/Dockerfile
         ports:
-            - "1337:1337"
+            - "5000:5000"
```

Comment:
Is this needed? The Dockerfile doesn't expose any ports and I don't know if it is used or not.